processing/src/main/java/org/apache/druid/query/aggregation/mean/DoubleMeanAggregatorFactory.java

```diff
@@ -152,6 +152,8 @@ public Object deserialize(Object object)
       return DoubleMeanHolder.fromBytes(StringUtils.decodeBase64(StringUtils.toUtf8((String) object)));
     } else if (object instanceof DoubleMeanHolder) {
       return object;
+    } else if (object instanceof byte[]) {
```

Comment:
can we make byte[] be  the firs if check in both  places ?

Reply:
But DoubleMeanHolder in both places too. Maybe we will replace only String in deserialize(..)?

Reply:
I meant, just move the byte[] check as being the first check in if-statement chain as I am expecting that would be the case most of the time.